bitcash/network/transaction.py

```diff
@@ -9,8 +9,8 @@ def __init__(self, txid, block, amount_in, amount_out, amount_fee):
         self.txid = txid
         self.block = block
 
-        if amount_in != amount_out + amount_fee:
-            raise ArithmeticError("the amounts just don't add up!")
+        if amount_in != amount_out + amount_fee:  # pragma: no cover
```

Comment:
Seems like a cheesy way to increase code coverage :).

Reply:
I couldn't see a way of reaching this exception, am I wrong? Maybe we can remove this altogether?

Otherwise I will revert this commit.

Reply:
As I am not 100% sure this error is never reached, I reverted this commit for the time being